source/backend/dal/Repositories/PropertyRepository.cs

```diff
@@ -381,6 +381,7 @@ public HashSet<long> GetMatchingIds(PropertyFilterCriteria filter)
         {
             var query = Context.PimsProperties.AsNoTracking();
```

Comment:
@FuriousLlama I feel like I'm missing something obvious, how could this possibly be working if there are no includes on the PimsProperties query?

Reply:
Afaik this means, loading all properties. The individual blocks below make sure to add conditionals